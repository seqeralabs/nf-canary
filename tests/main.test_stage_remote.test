nextflow_process {

    name "Test Process TEST_STAGE_REMOTE"
    script "main.nf"
    process "TEST_STAGE_REMOTE"

    test("Should run without failures") {

        when {

            params {
              remoteFile = "${baseDir}/tests/testfile.txt"
            }

            process {
                """
                input[0] = Channel.fromPath(params.remoteFile)
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
