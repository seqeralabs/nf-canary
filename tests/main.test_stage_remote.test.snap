{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    ""
                ]
            }
        ],
        "timestamp": "2023-10-04T12:00:49.001095"
    }
}